### PR TITLE
Update language_versions.md

### PR DESCRIPTION
**Pull request by tbrowder**
Start a mini-HOWTO on handling future versions

### CODE REVIEW

File: docs/language_versions.md [hunk 1]
```diff
@@ -3,6 +3,32 @@
 For review, nothing set in sand yet (and these things should never be set in
 stone; we need to keep learning and adapting them over time).
 
+## Adding the tooling structure to start a future version
+
+    TO BE ADDED
```
Comment (JJ) [inline docs/language_versions.md:8]: You mean later on in this same PR, or when it's merged?

File: docs/language_versions.md [hunk 2]
```diff
@@ -3,6 +3,32 @@
...
+    
+## Adding new features or deprecating features for a future version
+
+For example we use the upcoming new version '6.e' for which the tooling structure is in place
+and entries will be in the following directories an files:
```
Comment (JJ) [inline docs/language_versions.md:13]: typo: *and*

**Review by JJ — COMMENTED** [review]
So far it looks good, but maybe you'd want to mark it as draft so that it's not accidentally merged?

**Comment by tbrowder** [comment]
Good idea, so done.

**Comment by tbrowder** [comment]
Closing this PR: changes will be proposed in a new file.

**Pull request closed without merging**